platform-sdk/swirlds-crypto-tss/src/main/java/com/swirlds/crypto/tss/impl/TssServiceImpl.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.crypto.tss.impl;
+
+import com.swirlds.crypto.signaturescheme.api.PairingPrivateKey;
+import com.swirlds.crypto.signaturescheme.api.PairingPublicKey;
+import com.swirlds.crypto.signaturescheme.api.PairingSignature;
+import com.swirlds.crypto.signaturescheme.api.SignatureSchema;
+import com.swirlds.crypto.tss.TssMessage;
+import com.swirlds.crypto.tss.TssParticipantDirectory;
+import com.swirlds.crypto.tss.TssPrivateShare;
+import com.swirlds.crypto.tss.TssPublicShare;
+import com.swirlds.crypto.tss.TssService;
+import com.swirlds.crypto.tss.TssShareSignature;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * A prototype implementation of the TssService.
+ * future-work:Complete this implementation.
+ */
+public class TssServiceImpl implements TssService {
```

Comment:
Do we really want to have this class under src/main/java instead of src/test/java?

Reply:
As this is a prototype implementation of the TssService and the intention is to complete this implementation, I think the best place is here.